shared/src/main/scala/drt/shared/api/Arrival.scala

```diff
@@ -0,0 +1,170 @@
+package drt.shared.api
+
+import drt.shared.CrunchApi.MillisSinceEpoch
+import drt.shared.MilliTimes.oneMinuteMillis
+import drt.shared.Terminals.Terminal
+import drt.shared._
+import upickle.default.{ReadWriter, macroRW}
+
+import scala.collection.immutable.NumericRange
+import scala.util.matching.Regex
+
+case class ArrivalSuffix(suffix: String)
```

Comment:
FlightCodeSuffix might be a better name.

Reply:
Lovely suggestion thanks